manager/app/src/main/cpp/ksu.cc

```diff
@@ -55,7 +55,10 @@ int get_version() {
     int32_t version = -1;
     int32_t lkm = 0;
     ksuctl(CMD_GET_VERSION, &version, &lkm);
-    if (!is_lkm && lkm != 0) {
+    if (!is_lkm && lkm == 1) {
+    int32_t flags = 0;
+    ksuctl(CMD_GET_VERSION, &version, &flags);
```

Comment:
- Missing ending brace for the if-statement.
- Misleading indentation in L59 and L60.